content/5_Plat/RURQ.mdx

```diff
@@ -153,48 +322,43 @@ situations might take place after any update:
 Given the mechanics behind the `push_down` function, all you need is a regular
 range-sum segment tree to solve the question.
 
+### Implementation
+**Time Complexity:** $\mathcal{O}(N + Q\log{N})$
+
+<LanguageSection>
+<CPPSection>
+
 ```cpp
 #include <bits/stdc++.h>
 using namespace std;
 using ll = long long;
 
-/**
- * Represents the type of lazy update being done.
- * NONE = if there's no lazy update to be performed.
```

Comment:
also this doc comment

Reply:
good point idk why i got rid of that lmao